index.html

```diff
@@ -1,19 +1,68 @@
 <!doctype html>
 
 <html lang="en" dir="ltr">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <title>LXD UI</title>
-    <link rel="shortcut icon" href="/assets/img/favicon-32x32.png" type="image/x-icon">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <script>const global = globalThis;</script>
-  </head>
-  <body>
+  <title>LXD UI</title>
+  <link rel="shortcut icon" href="/assets/img/favicon-32x32.png" type="image/x-icon">
+  <link rel="manifest" href="/manifest.json" id="manifest">
 
-    <div id="app"></div>
-    <script type="module" src="/src/index.tsx"></script>
+  <script>const global = globalThis;
+
+  </script>
+</head>
+
+<body>
+
+  <div id="app"></div>
+  <script type="module" src="/src/index.tsx"></script>
+
+</body>
 
-  </body>
 </html>
+
+<script>
+
+  var screenshot = window.location.origin + "/assets/img/LXD-screenshot.png";
+
+  const manifestElement = document.getElementById("manifest");
+  const dynamicManifest = JSON.stringify({
+    "short_name": "LXD",
+    "name": "LXD-UI",
+    "icons": [
+      {
+        "src": window.location.origin + "/assets/img/canonical-lxd-512.png",
+        "type": "image/png",
+        "sizes": "512x512"
+      }
+    ],
+    "id": "LXDID",
+    "start_url": window.location.origin,
+    "background_color": "#E95420",
+    "display": "standalone",
+    "scope": "/",
```

Comment:
```suggestion
    "scope": window.location.origin,
```
I think this should also be the main url.